cli/azd/resources/starter/.azdo/pipelines/azd-up.yml

```diff
@@ -0,0 +1,45 @@
+trigger:
+  # Run when commits are pushed to mainline branch (main or master)
+  # Set this to the mainline branch you use
+  - main
+  - master
+
+# Example Azure Pipeline workflow to deploy to Azure using azd
+# To configure required secrets for connecting to Azure, simply run `azd pipeline config --provider azdo`
+
+pool:
+  vmImage: ubuntu-latest
+
+# Use azd provided container image that has azd, az, multi-language build tools pre-installed.
+container: mcr.microsoft.com/azure-dev-cli-apps:latest
+
+steps:
+  - pwsh: |
```

Comment:
As an aside, why do we have to use az CLI for auth in Azure Pipelines? I realize we do it in templates today but just out of curiosity...

Reply:
I think this is tracked in #1126.